validator/tools/sanitize_dashboards.js

```diff
@@ -0,0 +1,42 @@
+const { DEFINITIONS_DIR, DASHBOARD_FILE_NAME_SUFFIX, FILE_ENCODING} = require('./props');
+const {readdir} = require('fs').promises;
+const fs = require('fs');
+const path = require('path');
+
+(async () => {
+    const folderDefinitions = await readdir(DEFINITIONS_DIR, {withFileTypes: true})
+    for (let folderDefinition of folderDefinitions){
+        if(folderDefinition.isDirectory()){
+            const files = await readdir(path.resolve(DEFINITIONS_DIR, folderDefinition.name), {withFileTypes: true})
+            for(let file of  files) {
+                if(file.name.includes(DASHBOARD_FILE_NAME_SUFFIX)){
+                    let filePath = path.resolve(DEFINITIONS_DIR, folderDefinition.name, file.name)
+                    fs.readFile(filePath, FILE_ENCODING, function(err, fileContent){
+                        if (err){
+                            console.error(`Error reading ${file.name} in ${folderDefinition.name}`)
+                            process.exit(1)
+                        }
+
+                        let newFileContent = fileContent
```

Comment:
It Will be great to have a test there for this to have context in the future, and we are not breaking anything 😅

Reply:
I think that we don't have tests for any of the validation, I could add for this and create a ticket for the others

Reply:
Yep, but this validation is a little bit complex 😂. But I agree to have a ticket to create all the test

Reply:
Totally 😅 I'll mention in the ticket that we might want to add some integration tests for that. Thanks for the feedback!